packages/backend/src/modules/dashboard/dashboard.model.ts

```diff
@@ -4,8 +4,9 @@ import { DashboardVisibility } from 'shared'
 const { ObjectId } = mongoose.Types
 
 const columnSchema = new mongoose.Schema({
-  _id: ObjectId,
```

Comment:
why this change?

Reply:
with "_id" mongo doesn't generate id, if you have another solution for this problem I'm open to it